packages/client/docusaurus/docs/javascript/02-guides/04-camera-and-microphone.mdx

```diff
@@ -55,6 +55,20 @@ call.camera.state.selectedDevice; // currently selected camera
 call.camera.state.selectedDevice$.subscribe(console.log); // Reactive value for selected device, you can subscribe to changes
 ```
 
+### Camera permissions
+
+In a browser, you can use the following API to check whether the user has granted permission to access the connected cameras:
+
+```typescript
+call.camera.state.hasBrowserPermission$.subscribe((value) => {
+  if (value) {
+    // User has granted permission
+  } else {
+    // User has denied or not yet granted permission
+  }
+});
```

Comment:
For consistency, can we add a getter for the static value as well? Like this:

<img width="912" alt="Screenshot 2023-11-10 at 09 04 10" src="https://github.com/GetStream/stream-video-js/assets/6690098/b84e7edf-9392-4fbf-94c6-c94dad3299ae">

Reply:
I explicitly didn't add one, as that would mean we'd need to run the permission querying during the SDK load time.
We also can't query in the getter, as they are supposed to run synchronously -> so, opted out of adding a getter :)

Do you maybe know a better way?

Reply:
ah, I see, it's like the `listDevices` -> we only want to query if the user cares about this information. Yeah, not not having a getter makes sense.

Reply:
Yeah, it works the same as `listDevices`.